### ISSUE TITLE
Processing compiler becomes unresponsive with alternating println() and errors

### ISSUE THREAD

**Issue by rhpenguins** [processing/processing#5714]
## Description
When there is an error and a message logged to the console alternating it seems to make the IDE unresponsive.

## Steps to Reproduce
This code will cause the error to happen:
```
for (int i=0; i < 1000000; i++) {
    System.err.println("ERROR");
    println("out");
}
```

## Your Environment
* Processing version: 3.4
* Operating System and OS version: MacOS High Sierra
* Other information:

**Comment by StanLepunK** [comment]
I think the problem came of the print call in a loop` for(){}` Processing try to print in consol a million time in a single frame...that's huge !
read note about that in description
https://processing.org/reference/println_.html

**Comment by dzaima** [comment]
> I think the problem came of the print call in a loop` for(){}` Processing try to print in consol a million time in a single frame...that's huge !
> read note about that in description
> https://processing.org/reference/println_.html

just the `println` or `System.err.println` alone work just fine, but the loop with only 100k iterations seems to break very quickly (and I've certainly had it break before with way less printing)

---

so _that's_ what's causing that. This has been annoying me for way too long but i could never reproduce it well :D

**Comment by sarisarah** [comment]
The error is just there to explain how the error being printed into the console makes the IDE freeze.

**Comment by benfry** [comment]
Should be fixed in 4.0 beta 3: https://github.com/processing/processing4/issues/338

(And locking this issue as a duplicate.)
